packages/@lwc/module-resolver/src/__tests__/resolve.modules.spec.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+import path from 'path';
+import { resolveModulesFromDir, resolveModules } from '../index';
+
+const FIXTURE_MODULE_ENTRIES = ['ns/cssEntry', 'ns/htmlEntry', 'ns/jsEntry'];
+
+describe('resolve modules', () => {
```

Comment:
A couple of other tests that might be relevant -

Via the `resolveModules` API, should I be able to override the configuration?  (just as a note: tried this locally, it doesn't appear to pass)
```
    it('with configuration overrides resolving to custom modules', () => {
        const moduleDir = path.join(__dirname, 'fixtures/custom-resolution');
        const modules = resolveModules({ rootDir: moduleDir, modules: [["custom-module", "custom/module-alt.js"]] });
        const specifiers = modules.map(m => m.specifier);
        const entries = modules.map(m => m.entry);
        expect(specifiers).toStrictEqual(["custom-module"]);
        expect(entries).toStrictEqual([path.join(moduleDir, "custom/module-alt.js")]);
    });
```

Using the programmatic API to resolve one or more modules without a configuration file.
```
    it('from api configuration ', () => {
        const modules = resolveModules({ modules: ["@lwc/engine"] });
        const specifiers = modules.map(m => m.specifier);
        expect(specifiers).toStrictEqual(['lwc']);
    });
```

Reply:
added